### PR TITLE
Update README (iOS only going fwd) + Bump Deps + Add support for 'release' published trigger

### PR DESCRIPTION
**Pull request by NicoHinderling**
- README and action.yml have been updated (now uses node20)
- package.json's dep versions have been bumped (tested it works)

### CODE REVIEW

File: dist/37.index.js
Comment (NicoHinderling) [inline dist/37.index.js:1]: the `npm run release` command created this file. not clear why it's needed but 🤷

File: src/upload.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ async function run(): Promise<void> {
     body: JSON.stringify(requestBody),
   });
 
-  const data = await response.json();
+  const data: any = await response.json();
```
Comment (NicoHinderling) [inline src/upload.ts:31]: tsc lint issue fix

File: src/inputs.ts [hunk 2]
```diff
@@ -66,12 +70,12 @@ function getInputs(): UploadInputs {
...
   // Required for PRs
   const refName = process.env.GITHUB_REF ?? '';
-  var prNumber = getPRNumber(refName);
+  let prNumber = getPRNumber(refName);
   if (refName.includes('pull') && !prNumber) {
     core.setFailed('Could not get prNumber for a PR triggered build.');
   }
   if (!prNumber) {
-    prNumber = eventFileJson?.number
+    prNumber = eventFileJson?.number;
```
Comment (NicoHinderling) [inline src/inputs.ts:74]: both tsc lint issue fixes

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "emerge-upload-action",
-  "version": "1.0.3",
+  "version": "1.1.0",
```
Comment (NicoHinderling) [inline package.json:3]: I can also do 1.0.4, but this feels like a significant bump

**Review by rbro112 — APPROVED** [review]

**Pull request merged**